vent/test/db.go

```diff
@@ -42,10 +42,11 @@ func destroySchema(db *sqldb.SQLDB, dbSchema string) error {
 }
 
 // NewTestDB creates a database connection for testing
-func NewTestDB(t *testing.T) (*sqldb.SQLDB, func()) {
+func NewTestDB(t *testing.T, database string) (*sqldb.SQLDB, func()) {
 	t.Helper()
 
-	cfg := config.DefaultFlags()
+	cfg := config.DefaultFlags(database)
```

Comment:
As I commented previously, there's no need to add the `database` parameter to the `DefaultFlags` method, because it is intended to return the default config values, which use postgres as database. You can override the config values as needed after getting the default flags, the same way we are doing for the consumer tests here: https://github.com/hiturria/bosmarmot/blob/develop/vent/service/consumer_test.go#L77